### ISSUE TITLE
Incorrect file name in destination

### ISSUE THREAD

**Issue by sasokol** [werf/werf#3180]
```
werf version
v1.2.4+fix15
```

werf.yaml:
```
image: postgres
from: postgres:11-alpine
git:
- add: /path/to/initdb.sh
  to: /docker-entrypoint-initdb.d/initdb.sh
- add: /path/to/initdb.sql
  to: /docker-entrypoint-initdb.d/initdb.sql_
```
In git repo:
```
md5sum /path/to/initdb.*
6f753af736213f76d9260bfe7d8db6df  ls-backend/config/initdb.sh
42f5d5463fb7ec6d8e87e6175db82c66  ls-backend/config/initdb.sql
```

In docker image:
```
md5sum /docker-entrypoint-initdb.d/initdb*
6f753af736213f76d9260bfe7d8db6df  /docker-entrypoint-initdb.d/initdb.sh 
42f5d5463fb7ec6d8e87e6175db82c66  /docker-entrypoint-initdb.d/initdb.sql
```

But I expect
```
md5sum /docker-entrypoint-initdb.d/initdb*
6f753af736213f76d9260bfe7d8db6df  /docker-entrypoint-initdb.d/initdb.sh 
42f5d5463fb7ec6d8e87e6175db82c66  /docker-entrypoint-initdb.d/initdb.sql_
```

**Comment by distorhead** [comment]
This is bug https://github.com/werf/werf/issues/1741

I think we could try to fix this issue before v1.2 becomes stable.

**Comment by alexey-igrychev** [comment]
fixed in #3508
